src/Components/ImagesTable/ImagesTable.tsx

```diff
@@ -531,6 +533,11 @@ const Row = ({
         <Td dataLabel="Target">
           {target ? target : <Target compose={compose} />}
         </Td>
+        {compose.blueprint_version && (
+          <Td dataLabel="Version">
+            <Badge isRead>{compose.blueprint_version}</Badge>
+          </Td>
+        )}
```

Comment:
makes sense,  `blueprint_version` exists only when a blueprint is chosen, but better safe than sorry.

Why we need a `N/A` though? if a blueprint is chosen, its images must have a version. no?

Reply:
The case where `blueprint_version` is present even when `selectedBlueprint` is not selected worries me a bit :)

Yeah, `N/A` should never be used, but you never know :stuck_out_tongue: but yeah, that might be extra caution, that might not be needed now :)

Reply:
> The case where blueprint_version is present even when selectedBlueprint is not selected worries me a bit :) 

One issue,  `selectedBlueprint`  needs to be propagated multiple  levels,  from the Table component to all of its leaves...  Is there a case that the `blueprint_version` is present and we want to hide it?

Reply:
It's more about making sure that we always have the header shown when we have data in the column.
Imagine a case where we don't have `selectedBlueprint` and we have `blueprint_version` set on one of the images. The table becomes invalid, because one of the rows has one extra column. That's pretty bad.

Also if we show the header, but one of the rows whould have the cols missing would be an issue, but you've correctly stated it should never happen.

Reply:
When you need to propagate an information from a distant parent to all the leaves, you can use `useContext` for that.

Reply:
Looking at the latest mocks, I found out that the version column needs to be visible also when all images displayed, and there's no chosen blueprint:

<img width="980" alt="Screenshot 2024-02-05 at 18 31 54" src="https://github.com/RedHatInsights/image-builder-frontend/assets/11807069/1c252c1c-bf74-4c4d-adf1-d054ab702839">

@ezr-ondrej I think we need to update the  composes endpoint, and adding blueprint_version for every compose, if exists. In this PR's scope, we can use `experimental` variable / feature flag to gate the version column, so it won't leak. 

@lavocatt, I like the idea for using `useContext`, WDYT about adding a context for `experimental` flag,  so it will be exposed on every component?

Reply:
That sounds a bit confusing to me especially when the version column is not next to the blueprint (image) name, so it might not be very obvious what the version mean in the general context.

Would it make sense to start with having the version column only for selected blueprint, or clear it out with UX first?